cl_dll/in_camera.cpp

```diff
@@ -1,4 +1,4 @@
-//========= Copyright � 1996-2002, Valve LLC, All rights reserved. ============
+//========= Copyright � 1996-2002, Valve LLC, All rights reserved. ============
```

Comment:
nice

Reply:
Epic Visual Studio Code. >:(

Thirdperson cam while spectating works, it's implemented differently and elsewhere.